documentation/modules/auxiliary/scanner/http/robots_txt.md

```diff
@@ -0,0 +1,51 @@
+## Description
+
+This module will detect robots.txt files on web servers and analize its content.
+This type of file can reveal interesting information about areas of the site that are not indexed.
```

Comment:
Lets change this line to include a description of the file's purpose as well... something like:
The `robots.txt` file is a file which is supposed to be honored by web crawlers and bots, as locations which are not to be indexed or specifically called out to be indexed.  This can be abused to reveal interesting information about areas of the site which an admin may not want to be public knowledge.